File: 01-iniciante/1010/1010.py
codigoP1, qtdP1, valorP1 = input().split()
codigoP1, qtdP1, valorP1 = int(codigoP1), int(qtdP1), float(valorP1)

codigoP2, qtdP2, valorP2 = input().split()
codigoP2, qtdP2, valorP2 = int(codigoP2), int(qtdP2), float(valorP2)

totalP1 = qtdP1 * valorP1
totalP2 = qtdP2 * valorP2
totalFinal = totalP1 + totalP2

print(f'VALOR A PAGAR: R$ {totalFinal:.2f}')

File: 01-iniciante/1010/1010.c
#include <stdio.h>

int main()
{
    int codigoP1, qtdP1,codigoP2, qtdP2;
    float valorP1,valorP2;
    
    scanf("%d %d %f",&codigoP1,&qtdP1,&valorP1);
    scanf("%d %d %f",&codigoP2,&qtdP2,&valorP2);

    float totalP1 = qtdP1 * valorP1;
    float totalP2 = qtdP2 * valorP2;
    float totalFinal = totalP1 + totalP2;

    printf("VALOR A PAGAR: R$ %.2f\n",totalFinal);

    return 0;
}